### PR TITLE
improved search style

### LINKED ISSUE #360: Use last search results before researching

### PR DESCRIPTION
**Pull request by matthias-luger**
fixes #360 
improved styling

### CODE REVIEW

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-361.flou21.de

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-361.flou21.de

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-361.flou21.de

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-361.flou21.de

**Review by Ekwav — CHANGES_REQUESTED** [review]
Please make the input the the suggestion text to be in one row

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-361.flou21.de

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-361.flou21.de

File: src/components/Search/Search.tsx [hunk 1]
```diff
@@ -85,48 +82,68 @@ function Search(props: Props) {
...
+    };
+
+    let listWidth = document.getElementById('search-input-group')?.offsetWidth ? document.getElementById('search-input-group')!.offsetWidth - 2 : "";
+
     return (
         <div className="search">
 
             <Form autoComplete="off">
-                <Form.Group>
+                <Form.Group style={{ display: "flex", justifyContent: "center", alignContent: "center", marginBottom: 0, borderBottomWidth: 0 }}>
```
Comment (Ekwav) [inline src/components/Search/Search.tsx:121]: make this a class

**Review by Ekwav — CHANGES_REQUESTED** [review]
Why is the text in the input bigger than the suggestions?
The search icon should have the same style and position than the preview images

**Comment by Ekwav** [comment]
The tests don't pass after this change

**Comment by Ekwav** [comment]
The tests don't pass after this change

**Comment by matthias-luger** [comment]
I guess then there wont be a search style improvement.
Sad, because it really looks good and improves the search result loading. 👎

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-361.flou21.de

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-361.flou21.de

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-361.flou21.de

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-361.flou21.de

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-361.flou21.de

**Pull request merged**